config/pathSetup.m

```diff
@@ -101,11 +63,48 @@
 %% Print out the paths
 disp(['------------------------------------------------'])
 disp(['Setting up the the following path parameters'])
-disp(['Local GitHub repository path = ' gitPath])
+disp(['Local GitHub repository path = ' autogenRootDir])
 disp(['Simulink model path          = ' mp.model_path])
 disp(['Test signals path            = ' mp.test_signals_path])
 disp(['VHDL Codegen path            = ' mp.ipcore_codegen_path])
 disp(['Driver Codegen path          = ' mp.driver_codegen_path])
 disp(['UI Codegen path              = ' mp.ui_codegen_path])
 disp(['Quartus path                 = ' mp.quartus_path])
```

Comment:
Are these mp.*_path variables used elsewhere any more? If not, we can easily update them to match our coding style guidelines right now. If they are still used elsewhere, we can defer the renaming if we want.

Reply:
They are still used when calling Python because the Python calls are now system calls, and thus need the path to the Python files.